tpc-h/README.md

```diff
@@ -0,0 +1,234 @@
+# Quickstart for TPC-H Benchmark Sample Data
+
+> TPC-H is a decision support benchmark. It consists of a suite of business-oriented ad hoc queries and concurrent data modifications. The queries and the data populating the database have been chosen to have broad industry-wide relevance. This benchmark illustrates decision support systems that examine large volumes of data, execute queries with a high degree of complexity, and give answers to critical business questions.
+>
+> - [TPC Benchmark™ H (TPC-H)](https://www.tpc.org/tpch/)
+
+**Step 1.** Initialize and start Spice
+
+```bash
+spice init tpch-quickstart
+```
+
+```bash
+cd tpch-quickstart
+spice run
+```
+
+**Step 2.** Connect the TPC-H Benchmark Sample Data
+
+Add `datasets` configuration below to `spicepod.yaml`
```

Comment:
Lets just make this a spicepod.

Reply:
Just to confirm - list full spicepod.yaml definition/content + update wording? Not creating/publishing a spicepod as it will be harder to update definition to enable acceleration and demonstrate acceleration benefits (last step):


>Observe query execution time decreased from **4.178523666** to **0.108190459** seconds.

Reply:
Synched w/ Aurash and agreed to keep as-is for now.